wetterdienst/dwd/radar/access.py

```diff
@@ -225,6 +225,8 @@ def _download_generic_data(url: str) -> Generator[RadarResult, None, None]:
 
     _, data = _download_generic_data_cached(url)
 
+    data.seek(0)
+
     # RadarParameter.FX_REFLECTIVITY
     if url.endswith(".tar.bz2"):
```

Comment:
We may add this one to the enumeration I have created for all types of archive endings.

Reply:
Thanks. Implemented with b6897ce62.